content/docs/about/events.md

```diff
@@ -10,13 +10,59 @@ Please raise a [GitHub issue](https://github.com/kubeflow/website/issues/new) if
 
 * [Google Cloud Next, San Francisco](https://cloud.withgoogle.com/next/sf), 9-11 April, 2019
   - [Undoing Human Bias at Scale With Kubeflow](https://cloud.withgoogle.com/next/sf/sessions?session=MLAI206): John Bohannon, Michelle Casbon
-* Kubeflow Contributor Summit 2019, Sunnyvale, CA, 12-13 March, 2019
+* [Kubeflow Contributor Summit 2019](https://docs.google.com/document/d/14Jr79aWVjsJg8xq38CLVE4rKRwM1_-gHLuxhGAhrGTI/edit?usp=sharing), Sunnyvale, CA, 12-13 March, 2019
+  - Kubeflow and the ML Landscape: Jeremy Lewi, Google
```

Comment:
You could consider making this list into a table using the syntax:
```md
Team | Person | Slides / Other
--- | --- | ---
foo | bar | [slides](https://link/here)
```

Reply:
Possible, but I was trying to go by the format used in the list -- would like to leave as is and fix all events with several talks in one refactor.

Reply:
Sounds good!